docs/src/assets/landing-page/nav-items.js

```diff
@@ -9,13 +9,128 @@ import {
 } from '@quasar/extras/mdi-v6'
 import { fasCubes } from '@quasar/extras/fontawesome-v5'
 
+const gettingStartedNavItem = {
+  label: 'Getting Started',
+  subMenu: [
+    {
+      label: 'Installation',
+      subMenu: [
+        {
+          label: 'Pick Quasar Flavour',
+          path: 'start/pick-quasar-flavour'
+        },
+        {
+          label: 'Quasar CLI',
+          path: 'quasar-cli'
+        },
+        {
+          label: 'UMD / Standalone',
+          path: 'start/umd'
+        },
+        {
+          label: 'Vue CLI Plugin',
+          path: 'start/vue-cli-plugin'
+        },
+        {
+          label: 'Vite Plugin',
+          path: 'start/vite-plugin'
```

Comment:
This brings to nowhere, we probably need to rebase on upstream as we miss this page in our version